analyzers/tests/SonarAnalyzer.UnitTest/Rules/ObsoleteAttributesNeedExplanationTest.cs

```diff
@@ -18,30 +18,40 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using SonarAnalyzer.Rules.CSharp;
+using CS = SonarAnalyzer.Rules.CSharp;
+using VB = SonarAnalyzer.Rules.VisualBasic;
 
-namespace SonarAnalyzer.UnitTest.Rules
+namespace SonarAnalyzer.UnitTest.Rules;
+
+[TestClass]
+public class ObsoleteAttributesNeedExplanationTest
 {
-    [TestClass]
-    public class ObsoleteAttributesNeedExplanationTest
-    {
-        private readonly VerifierBuilder builder = new VerifierBuilder<ObsoleteAttributesNeedExplanation>();
+    private readonly VerifierBuilder csBuilder = new VerifierBuilder<CS.ObsoleteAttributesNeedExplanation>();
+    private readonly VerifierBuilder vbBuilder = new VerifierBuilder<VB.ObsoleteAttributesNeedExplanation>();
```

Comment:
..as we use it in the rest (will require same changes below)
```suggestion
    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.ObsoleteAttributesNeedExplanation>();
    private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.ObsoleteAttributesNeedExplanation>();
```